ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/AgentUtils.java

```diff
@@ -472,6 +472,11 @@ public static Map<String, String> constructToolParams(
         if (toolSpecConfigMap != null) {
             toolParams.putAll(toolSpecConfigMap);
         }
+        toolParams.put("action_input", actionInput);
```

Comment:
`action_input` looks like coupled with code logic. Change to more readable name ?

Reply:
Changed.